src/app/components/feed/connection-config.factory.js

```diff
@@ -24,7 +24,7 @@
     return function(pluginHandle, wantedInit, jsep, ok) {
       var current = {};
       var requested = null;
-      var wanted = {audio: true, video: true};
+      var wanted = {audio: false, video: true};
```

Comment:
It seems like it doesn't make a difference, but with audio: false I get "Connection configured Object {audio: false, video: true}" two times in the console, while with audio: true only one time. When I tested I got some configuration error and thought this was the cause but it isn't. So maybe I should revert this change..